content/api/cli.md

```diff
@@ -1,8 +1,264 @@
 ---
-title: Using the CLI
+title: CLI
 ---
-> Cli call
-> Cli options
-> --config
+## Overview
 
-> see also [[Using the configuration]]
+`webpack` provides a command line interface(cli) to configure and interact with your build.
+This is mostly useful in case of early prototyping, profiling, writing npm scripts or personal customization of the build.
+For proper usage and easy distribution of this configuration, webpack can be configured with `webpack.config.js`.
+Any parameters sent to the CLI will map to a corresponding parameter in the config file.
+
+## Installation
+
+``` sh
+$ npm install webpack -g
+```
+
+The `webpack` command is now available globally.
```

Comment:
I would mention how to use webpack locally as a tip. Basically

1. `npm install webpack` within a project.
2. `npm bin` to figure out where it installed.
3. `node_modules/.bin/webpack` or similar works after that.

Given how npm works, webpack installed to project is available through npm scripts so you can use `"start": "webpack-dev-server"` and such commands without global installation.

I would go as far as to encourage users to prefer local install over global one given it's easier to manage.